code/datums/config/config_num.dm

```diff
@@ -0,0 +1,15 @@
+/decl/config/num
+	abstract_type = /decl/config/num
+	config_flags = CONFIG_FLAG_NUM | CONFIG_FLAG_HAS_VALUE
+	default_value = 0
+	var/min_value = -(INFINITY)
+	var/max_value = INFINITY
+	var/rounding
+
+/decl/config/num/sanitize_value()
+	..()
+	value = clamp(value, min_value, max_value)
+	if(isnull(rounding))
+		value = round(value)
```

Comment:
I think `rounding = null` should skip rounding entirely. This seems to have introduced a number of CI fails.